nilearn/plotting/displays.py

```diff
@@ -1216,6 +1221,9 @@ def add_graph(self, adjacency_matrix, node_coords,
                               vmin=edge_vmin, vmax=edge_vmax,
                               **edge_kwargs)
 
+        if colorbar:
+            self._colorbar_show(ax, threshold=edge_threshold)
```

Comment:
> Suddenly you are passing some kind of axis object which is very surprising. "How can that even work" is how I felt until I realised that the .cmap and .norm attributes were set on the GlassBrainAxes instance just for the sole purpose of making _colorbar_show work.

By definition, the classes defined here with suffix "Axes" are supposed to be classes compatible with matplotlib Axes in order to be able to feed them to functions that are supposed to process Axes... So, in my sense, this hack makes sense in that context.

I mean, this hack is not nastier than the whole system already is.